ref/general/deploying-openshift.adoc

```diff
@@ -12,23 +12,22 @@
 :page-type: general
 = Deploying Open Liberty applications on OpenShift
 
-link:https://www.openshift.com/[Red Hat OpenShift] is a Kubernetes-based application platform.
-After you write your applications, you can containerize and deploy them to OpenShift to orchestrate and automate your containers.
-You can also implement the Open Liberty Operator to simplify the process of deploying and managing your applications in the cloud.
+link:https://www.openshift.com/[OpenShift] is a Kubernetes-based application platform.
+After you write an application, you can use OpenShift to build, containerize, and deploy your application so it's highly available and scalable.
+You can use the Open Liberty Operator to simplify the process of deploying and managing your applications in the cloud.
 
 == What is OpenShift?
-OpenShift is an open source application platform that is based on link:https://kubernetes.io/[Kubernetes].
-Kubernetes is an orchestration platform for containerized applications.
-After you develop and containerize your applications, your containers must be able to communicate with each other and scale as each service is needed.
-Kubernetes provides features that automate and manage containers to meet the needs of developers and ops teams.
+OpenShift is a Kubernetes-based application platform for container orchestration.
+After you develop and containerize your applications, your containers need to communicate with each other and scale as each service is needed.
+OpenShift provides features that automate and manage containers to meet the needs of developers and ops teams.
 
-OpenShift is a distribution of Kubernetes that takes the upstream Kubernetes project and adds fixes and helpful tooling.
-It is also thoroughly tested before every release.
-These additions support enterprise applications, and you can have peace of mind when you know that your application platform was thoroughly tested and debugged.
+OpenShift takes the upstream Kubernetes project and adds fixes and helpful tooling.
+It's also thoroughly tested before every release.
+These additions support enterprise applications, and you can have peace of mind knowing that your application platform was thoroughly tested and debugged.
 
 == Running applications on OpenShift
-One of the benefits of running your applications on OpenShift is that you can deploy them to a cloud platform, or to your current on-premises structure.
-You can use the link:https://docs.openshift.com/online/starter/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI] or link:https://github.com/openshift/odo[OpenShift command line tools] to develop your application.
+One of the benefits of running your applications on OpenShift is that you can deploy them to a cloud-hosted IaaS, or to your current on-premises structure.
```

Comment:
Should Infrastructure as a service (IaaS) be spelled out on first mention? I'm not sure